hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -249,27 +257,36 @@ public void handleRound(
                             .equals(Instant.EPOCH);
                     case BLOCKS, BOTH -> blockStreamManager.pendingWork() == GENESIS_WORK;
                 };
+        boolean systemTransactionsDispatched = false;
         if (isGenesis) {
             final var genesisEventTime = round.iterator().next().getConsensusTimestamp();
             logger.info("Doing genesis setup before {}", genesisEventTime);
             systemTransactions.doGenesisSetup(genesisEventTime, state);
+            systemTransactionsDispatched = true;
```

Comment:
This change will be removed once this PR is merged https://github.com/hiero-ledger/hiero-consensus-node/pull/18562